powersync-ja/powersync-js
# feat(react): add usePowerSyncStatus hook

## Description
Add usePowerSyncStatus hook to react package

## Work Done
* Added `usePowerSyncStatus`
* Added testing to react package
* Updated demo to use new hook
* Removed unused imports from web SDK tests

## How to test
* Load `react-supabase-todolist` and make sure Wifi icon is changing is expected (i.e. online when connected and offline when disconnected)
